code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -115,8 +115,6 @@
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/dual/simple
 	sawn_desc = "Short and concealable, terribly uncomfortable to fire, but worse on the other end."
 	fire_sound = 'sound/f13weapons/caravan_shotgun.ogg'
-	extra_damage = 3.2
-	extra_penetration = 0.1
```

Comment:
The added penetration of double barrel shotguns was meant to be a primary gameplay gimmick, due to their low capacity. But after the spawn tweaks, maybe we can leave it at 0

Reply:
Putting a note on it, but leaving it for this PR. Feel free to add it back immediately.
I intend to follow this PR up, however, with the spawn adjustments.